### DISCUSSION TITLE
Stop video at predefined position

### DISCUSSION THREAD

**Discussion by MiroLiebschner** [jhomlala/betterplayer#547] [Q&A]
Usecase: 
The video should stop at defined second or millisecond (so a PushUp with a question can be showed).

How I do it right now:
In didChangeDependencies() I check 2 times per second if the second is the second I'm looking for. 

Problem: 
To be able to stop at predefined frame I want to stop at predefined millisecond. 
My solution would need me to check every millisecond which seems to be a bad solution performance wise.

Is there any other option I'm overlooking? 

Thanks guys & nice package!

**Comment by jhomlala** [comment]
There's no functionallity in Better Player which allows to stop video at predefined time. Anyway checking video time in didChangeDependencies may be not optimal. You should listen to the progress event and setup behavior if video position passed given time.
